var/spack/repos/builtin/packages/tasmanian/package.py

```diff
@@ -30,7 +30,89 @@ class Tasmanian(CMakePackage):
     ApproximatioN is a robust library for high dimensional integration and
     interpolation as well as parameter calibration."""
 
-    homepage = "http://tasmanian.ornl.gov"
-    url      = "http://tasmanian.ornl.gov/documents/Tasmanian_v5.0.zip"
+    homepage = 'http://tasmanian.ornl.gov'
+    url      = 'https://github.com/ORNL/TASMANIAN/archive/v5.1.tar.gz'
 
-    version('5.0', '4bf131841d786033863d271739be0f7a')
+    version('develop', branch='master',
+            git='https://github.com/ORNL/TASMANIAN.git')
```

Comment:
Now that #3161 has been merged, you can add `git` as a package-level URL right alongside `url`. Since you use the `branch` parameter, Spack will know that it needs to use `git` for this version.

Reply:
I'll try to sync with upstream, but currently I am getting the same error, spack looks for `homepage` which is not the `git` repo.